src/main/java/org/kainos/ea/api/JobService.java

```diff
@@ -12,7 +13,14 @@
 
 
 public class JobService {
-    private JobDao jobDao = new JobDao();
+    private JobDao jobDao;
+    private DatabaseConnector databaseConnector;
+    public JobService(JobDao jd, DatabaseConnector dbc) {
+        databaseConnector = dbc;
+        jobDao = jd;
+    }
```

Comment:
Remove the database connector from the constructor.

Reply:
> Will fix this and the other job related issues in pr 002